src/App.js

```diff
@@ -1,25 +1,182 @@
-import logo from './logo.svg';
 import './App.css';
+import firebase from './firebase.js';
+import { getDatabase, ref, onValue, update, get } from 'firebase/database';
+import { useEffect, useState } from 'react';
+
+import Header from './Header';
+import Footer from './Footer';
+
+const App = () => {
+  const [userChoice, setUserChoice] = useState(null)
+  const [computerChoice, setComputerChoice] = useState(null)
+  const [result, setResult] = useState(null)
+  const [userValue, setUserValue] = useState(0)
```

Comment:
Great job using useState! Something I started exploring with my project was using objects with state to track multiple values which maybe something you could use. Possibly taking the user and computer value into one state.